www/js/metrics.js

```diff
@@ -1178,6 +1178,10 @@ angular.module('emission.main.metrics',['nvd3', 'emission.services', 'ionic-date
   }, 1)
 
   $scope.doRefresh = function() {
+    ClientStats.addEvent(ClientStats.getStatKeys().OPENED_APP).then(
```

Comment:
Is this really where you want to add this stat?
As far as I can tell, `doRefresh` is called when the user clicks the refresh button.
That doesn't really correspond to app open, does it?